### ISSUE TITLE
basil scripts fail to run if story editor is open

### ISSUE THREAD

**Issue by trych** [basiljs/basil.js#94]
If the story editor is open, basil scripts will fail to run.

The error happens in the setup stage, when `currPage` is getting defined. It occurs in line `990`:

`currPage = app.activeWindow.activePage;`

'Object does not support property or method active page'. Apparently it is not sufficient in the previous check to see if `currDoc.windows.length`, but there needs to be an additional check, if `activePage` can be accessed.

**Comment by fabianmoronzirfas** [comment]
So should we `close` it for the user?  
http://yearbook.github.io/esdocs/#/InDesign/StoryWindow/close  
Or just break if the returned value is `instanceof StoryWindow`?  
http://yearbook.github.io/esdocs/#/InDesign/Application/activeWindow  

The line is this one I guess https://github.com/basiljs/basil.js/blob/master/src/includes/environment.js#L161

**Comment by trych** [comment]
No, we should not close it, I think we should interfere with the user's setup as little as possible. Surely there must be a way to retrieve the `activePage` from the frontmost `LayoutWindow`, even when there is a `StoryWindow` open?

Yes, that was the line IIRC.

**Comment by b-g** [comment]
@trych good luck! If it is too painful just use the less dirty workaround possible to fix the bug.

**Comment by fabianmoronzirfas** [comment]
@trych What is the state of this? Can I assign you to it?
